src/utils.ts

```diff
@@ -27,3 +29,33 @@ export const deriveOvenAddress = async (operation: TransactionWalletOperation):
 
   return ovenAddress
 }
+
+/**
+ * Convert an interest rate per period into an APY.
+ */
+export const interestRateToApy = async (interstRatePerPeriod: BigNumber): Promise<BigNumber> => {
+  const one = new BigNumber(1_000_000_000_000_000_000)
```

Comment:
Instead of hard-coded (but nicely formatted) values like this, can we always denote values like this with a mantissa? 

Something like this 
```const one = new BigNumber(10).pow(18)```